<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="firstSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="secondSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="NodeTreeView" name="treeView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QGraphicsView" name="view">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QWidget" name="deviceInterfaceWidgets" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QComboBox" name="deviceInterfaceComboBox"/>
            </item>
            <item>
             <widget class="QWidget" name="deviceInterfaceComponents" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>250</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="debugMessagesTabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImportData">
     <property name="title">
      <string>Import Data</string>
     </property>
     <addaction name="actionImportScenario"/>
     <addaction name="actionImportShip"/>
     <addaction name="actionImportController"/>
     <addaction name="actionImportImage"/>
     <addaction name="actionImportObject"/>
    </widget>
    <widget class="QMenu" name="menuOpenFile">
     <property name="title">
      <string>Open File</string>
     </property>
     <addaction name="actionOpenScenario"/>
     <addaction name="actionOpenShip"/>
     <addaction name="actionOpenController"/>
     <addaction name="actionOpenImage"/>
     <addaction name="actionOpenObject"/>
    </widget>
    <addaction name="actionLoadScenario"/>
    <addaction name="separator"/>
    <addaction name="menuOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionImportPackage"/>
    <addaction name="menuImportData"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionSimulationAutoRestart"/>
    <addaction name="actionFitAllOnStart"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionLoadScenario">
   <property name="text">
    <string>Load Scenario</string>
   </property>
  </action>
  <action name="actionImportScenario">
   <property name="text">
    <string>Scenario</string>
   </property>
  </action>
  <action name="actionImportShip">
   <property name="text">
    <string>Ship</string>
   </property>
  </action>
  <action name="actionImportController">
   <property name="text">
    <string>Controller</string>
   </property>
  </action>
  <action name="actionImportPackage">
   <property name="text">
    <string>Import Package</string>
   </property>
  </action>
  <action name="actionImportImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionOpenScenario">
   <property name="text">
    <string>Scenario</string>
   </property>
  </action>
  <action name="actionOpenShip">
   <property name="text">
    <string>Ship</string>
   </property>
  </action>
  <action name="actionOpenController">
   <property name="text">
    <string>Controller</string>
   </property>
  </action>
  <action name="actionOpenImage">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="actionOpenObject">
   <property name="text">
    <string>Object</string>
   </property>
  </action>
  <action name="actionImportObject">
   <property name="text">
    <string>Object</string>
   </property>
  </action>
  <action name="actionSimulationAutoRestart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Restart</string>
   </property>
  </action>
  <action name="actionFitAllOnStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit all on start</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NodeTreeView</class>
   <extends>QTreeView</extends>
   <header>nodetreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
